clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/EstimateDataRecoveryTimeCommand.java

```diff
@@ -0,0 +1,69 @@
+package com.linkedin.venice.datarecovery;
+
+/**
+ * EstimateRecoveryTimeCommand contains the details of a request for estimating the recovery time of a store.
+ * We expect the command to comply with the following contract:
+ *
+ * Input:
+ *    admin-tool.sh --list-store-push-info --url <url>--store <store_name> --cluster <source_fabric>
+ * Output:
+ *    success: link_to_running_task
+ *    failure: failure_reason
+ */
```

Comment:
This does not hold, because it is different from `StoreRepushCommand` where it spawns a child process to do the real work. The admin-tool process does the actual work by itself for this command.